frontend/src/components/place/PlaceSidebarInfo.vue

```diff
@@ -26,10 +26,34 @@
                         {{ tag.name }}
                     </span>
                 </div>
+
                 <div class="place-sidebar__worktime">
                     <i class="place-sidebar__icon far fa-clock" />
-                    <span class="worktime-info--red">Закрыто до 15:00</span>
+                    <div class="level">
+                        <div class="level-left">
+                            <span v-if="isOpen() === true" class="worktime-info--green">Open now</span>
```

Comment:
For service phrases of an application use localization constants as it has done in the login page

Reply:
Done.